common/aws_io_params.yaml

```diff
@@ -588,3 +588,23 @@ i7ie.ALL:
   read_bandwidth: 3422623232
   write_iops: 119327
   write_bandwidth: 1526442410
+i7i.large:
+  read_iops: 82265
```

Comment:
From https://docs.aws.amazon.com/ec2/latest/instancetypes/so.html#so_instance-store , I see that i7i.large has 75,000 / 41,250 (100% random read IOPS / Write IOPS) - and our values are higher. Does that make sense?

Reply:
I think it's probably an error.